### PR TITLE
Fix/properties and children in builtin languages

### PR DESCRIPTION
**Pull request by odzhychko**

### CODE REVIEW

File: model-api/src/commonTest/kotlin/org/modelix/model/api/BuiltinLanguagesTest.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+package org.modelix.model.api
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class BuiltinLanguagesTest {
+
+    @Test
+    fun propertiesAreAdded() {
```
Comment (languitar) [inline model-api/src/commonTest/kotlin/org/modelix/model/api/BuiltinLanguagesTest.kt:25]: Maybe something like "allPropertiesAreAvailableWithoutPriorAccess"?

**Review by languitar — DISMISSED** [review]
For the two newly added tests, an explanatory comment for why this seemingly simple condition is being tested could help

File: model-api/src/commonTest/kotlin/org/modelix/model/api/BuiltinLanguagesTest.kt (hunk 1 above)
Comment (odzhychko) [reply model-api/src/commonTest/kotlin/org/modelix/model/api/BuiltinLanguagesTest.kt]: Done. Also added more detailed comments in the test code.

**Comment by languitar** [comment]
Is it intended that this PR now includes a few of @mhuster23 's commits from his draft PR?

**Comment by odzhychko** [comment]
> Is it intended that this PR now includes a few of @mhuster23 's commits from his draft PR?

No that was an accident. I already removed them.

**Review by languitar — APPROVED** [review]

**Pull request merged**